### PR TITLE
[TW-738] Verified teams

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
Verified teams will have a verified badge next to their team name in the Public API Network, and other occurrences of their team name. Currently, API publishers can't request the verified badge.

* Added a section about verified teams written from the API consumer perspective.
* Added a screenshot, showing the verified badge in the Public API Network.
* Explained other areas where the verified badge displays.

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Review by gitThingsDone — APPROVED** [review]
Looks good overall, it would be great if we can update the screenshot too.

File: src/pages/docs/getting-started/first-steps/exploring-public-api-network.md [hunk 1]
```diff
@@ -54,6 +54,14 @@ Check out some useful collections for getting started learning about APIs, reque
 * [How to use the Postman Console](https://www.postman.com/postman/workspace/postman-team-collections/collection/1559645-9349429e-3744-467b-a127-e3881f0dffc8?ctx=documentation)
 * [Postman Echo](https://www.postman.com/postman/workspace/published-postman-templates/documentation/631643-f695cab7-6878-eb55-7943-ad88e1ccfd65?ctx=documentation)
 
+## About verified teams in the Public API Network
+
+Verified teams in Postman have a verified badge next to their team name in the Public API Network. A verified badge means Postman has verified the team's identity as the authentic organization they represent, helping you discover authentic, useful, and trusted APIs.
+
+<img alt="Postman verified team" src="https://assets.postman.com/postman-docs/v10/verified-team-postman-v10.jpg" width="400px" />
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/first-steps/exploring-public-api-network.md:61]: @gitThingsDone Yes, I like that better. I'll work on this change. Thanks!

[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/first-steps/exploring-public-api-network.md]: This is now updated.

**Pull request merged**